flake.nix

```diff
@@ -0,0 +1,25 @@
+{
+  description = "A very basic flake";
+
+  outputs = { self, nixpkgs }: {
+
+    packages.x86_64-linux.tinycc =
+      with import nixpkgs { system = "x86_64-linux"; };
+      stdenv.mkDerivation {
+        name = "tinycc";
+        src = self;
+        nativeBuildInputs = [ perl texinfo which ];
+        configureFlags = [
+          "--cc=cc"
+          "--crtprefix=${lib.getLib stdenv.cc.libc}/lib"
+          "--sysincludepaths=${lib.getDev stdenv.cc.libc}/include:{B}/include"
+          "--libpaths=${lib.getLib stdenv.cc.libc}/lib"
+          # build cross compilers
+          "--enable-cross"
```

Comment:
It will make more sense to use Nix to handle the cross compilation in the future by setting `system` and generating packages for every single architecture, like `packages.riscv64-linux.tinycc`, etc. The way it is now means that cross-compilers are unnecessarily built, and that the code will compile both natively and cross. But this is simpler for now, especially if you want to understand what is happening with little abstraction early on.

Reply:
@morganthomas That's correct, I misunderstood the purpose. However, we may want to introduce a new `stdenv` and package set of our own for compiling whatever it is we intend to compile with tinycc. Then we could have a flake which lets you `nix build .#pkgsTiny.packageName` and automates a lot of things for us, especially for CI, as we could build and report failure of the compilation for every single package in our set.